appointment/views.py

```diff
@@ -219,7 +220,7 @@ def appointment_request(request, service_id=None, staff_member_id=None):
         'available_slots': available_slots,
         'date_chosen': date_chosen,
         'locale': get_locale(),
-        'timezoneTxt': get_timezone_txt(),
+        'timezoneTxt': get_current_timezone_name(),
```

Comment:
This is a better choice than the custom get_timezone_txt() function, which is limited...